lrauv_ignition_plugins/test/test_mission_depth_vbs.cc

```diff
@@ -53,23 +53,22 @@ TEST_F(LrauvTestFixture, DepthVBS)
         lrauvRunning);
   });
 
+  int maxIterations{28000};
   int maxSleep{100};
   int sleep{0};
-  for (; sleep < maxSleep && lrauvRunning; ++sleep)
+  for (; sleep < maxSleep && lrauvRunning && this->iterations < maxIterations; ++sleep)
   {
     igndbg << "Ran [" << this->iterations << "] iterations." << std::endl;
     std::this_thread::sleep_for(1s);
   }
   EXPECT_LT(sleep, maxSleep);
-  EXPECT_FALSE(lrauvRunning);
+  ASSERT_LT(maxIterations, this->tethysPoses.size());
```

Comment:
I realize this line is not new either, but I want to check reader understanding. Is this assertion to make sure it has run for at least `maxIterations`? In that case, maybe it should be renamed to something like targetIterations, because at first I thought there's a typo, asserting a "max" is less than something. Same for other files.

Reply:
True, went with `targetIterations`:  ad14dbf